freud/order.pyx

```diff
@@ -255,11 +255,16 @@ cdef class Hexatic(_PairCompute):
     Args:
         k (unsigned int, optional):
             Symmetry of order parameter. (Default value = :code:`6`).
+        weighted (bool, optional):
+            Determines whether to use neighbor weights in the computation of
+            spherical harmonics over neighbors. If enabled and used with a
+            Voronoi neighbor list, this results in the Minkowski Structure
+            Metrics :math:`q'_l`. (Default value = :code:`False`)
```

Comment:
@gabriele16 What's the best symbol to use here? I typically use `\psi` to refer to hexatic / 2D and `q` to refer to Steinhardt-like 3D parameters, but I see some literature uses `q` for the 2D case.

Reply:
I decided to use `\psi'_k` for the 2D Minkowski Structure Metric, to match what we've done with Steinhardt, where `q'_l` is the 3D Minkowski Structure Metric.